tests/e2e/utils/workspace/TestWorkspaceUtil.ts

```diff
@@ -8,16 +8,18 @@
  * SPDX-License-Identifier: EPL-2.0
  **********************************************************************/
 
+import axios from 'axios';
+import { che } from '@eclipse-che/api';
```

Comment:
As far as I understood this dependency becomes from che,
and it means that for launching any tests we should build che previously.
That's why I strongly disagree with that.